test/views/RejectedNotificationErrorsViewSpec.scala

```diff
@@ -51,31 +52,48 @@ class RejectedNotificationErrorsViewSpec extends UnitViewSpec with Stubs with In
 
     "have correct title" in {
 
-      view.getElementById("title").text() mustBe messages("rejected.notification.title")
+      view().getElementById("title").text() mustBe messages("rejected.notification.title")
     }
 
     "have correct back link" in {
 
-      val backLink = view.getElementById("link-back")
+      val backLink = view().getElementById("link-back")
```

Comment:
can we extract default arg view render as variable and don't render it for each case ?

Reply:
yup

Reply:
Whats the harm?

Reply:
Rendering 10x allocates big string 10x and puts unnecessary GC pressure on tests. Unless we expect that view render with same parameters give use different outcomes.